public/locales/en-US/grafana.json

```diff
@@ -1346,6 +1346,10 @@
     "explore": {
       "title": "Explore"
     },
+    "external-group-sync": {
+      "title": "External group sync",
+      "subtitle": "Manage identity provider group mappings to Grafana user attributes"
+    },
```

Comment:
Since you added a string with translation please run `make i18n-extract`. Afterwards there should be changes in this file and in `public/locales/pseudo-LOCALE/grafana.json`.